addons/knowledge/models/knowledge_article.py

```diff
@@ -655,7 +655,10 @@ def move_to(self, parent_id=False, before_article_id=False, private=False):
             sequence = before_article.sequence
         else:
             # get max sequence among articles with the same parent
-            sequence = self._get_max_sequence_inside_parent(parent_id)
+            max_sequence_article = self.env['knowledge.article'].search(
```

Comment:
This one conflicts with work done when moving this at create/write level, so maybe we keep the other version and discuss on it if necessary ? (not saying mine is better because it is mine, just I prefer when things are done at lowest level possible)